packages/datagateway-search/cypress/integration/search/datafileSearch.spec.ts

```diff
@@ -32,7 +33,7 @@ describe('Datafile search tab', () => {
 
     cy.get('[aria-label="Submit search"]')
       .click()
-      .wait(['@datafiles', '@datafiles', '@datafilesCount'], {
+      .wait(['@searchDocuments', '@datafilesCount'], {
```

Comment:
Am I right in saying that the queries to /search/documents now cover all attempts to query individually for investigations, datasets and datafiles? If so, we should remove the intercepts for investigations, datasets and datafiles, since these are no longer required (see lines 16, 18 and 20)

Reply:
Actually, I forgot that individual count queries aren't made either. So all the intercepts between lines 13 and 20 can be removed (also make sure to remove any `cy.wait` calls we make to their respective aliases